assets/js/modules/pagespeed-insights/index.js

```diff
@@ -43,19 +48,40 @@ const {
 	setupComplete,
 } = getModulesData()[ 'pagespeed-insights' ];
 
+// @TODO: remove LegacyDashboardSpeed once all widgets have been migrated.
 if ( active && setupComplete ) {
 	// Add to main dashboard.
 	addFilter(
 		'googlesitekit.DashboardModule',
 		'googlesitekit.PageSpeedInsights',
-		createAddToFilter( <DashboardSpeed /> ),
+		createAddToFilter( <LegacyDashboardSpeed /> ),
 		45
 	);
+
 	// Add to dashboard-details view.
 	addFilter(
 		'googlesitekit.DashboardDetailsModule',
 		'googlesitekit.PageSpeedInsights',
-		createAddToFilter( <DashboardSpeed /> ),
+		createAddToFilter( <LegacyDashboardSpeed /> ),
+		45
+	);
+}
+
+if ( ! active || ! setupComplete ) {
```

Comment:
Nit-picking, but this could as well be in an `else` for the above.

Reply:
Updated.